src/main/resources/fabric.mod.json

```diff
@@ -10,7 +10,7 @@
   "contact": {},
   "license": "MIT",
   "icon": "assets/fallingleaves/icon.png",
-  "environment": "*",
+  "environment": "client",
```

Comment:
Shouldn't have `client` here but leave the `main` entrypoint. Just move everything in `ModInitializer` over to `ClientModInitializer`.

Reply:
The timing on this, I was just about to replace this pull request!
One thing it'll do is remove `main`/`ModInitializer` completely as there's absolutely no need for it.

Reply:
Oh yeah, I meant to say that you can't have both `client` and a `main` entrypoint. Well maybe you can I haven't tried but it makes no sense obviously.

Reply:
Ah, you can have both and both are called - I think `main` is called first.
`main` is called both on server and client, `client` only on client (duh) and `server`, well you know..
It can make sense to have both, e.g. if your mod is for both server and client and is configurable, you can put the config initialization stuff in `main`, so you don't have to duplicate your code in both the `client` and `server` entrypoints.

Reply:
I was confusing again: it doesn't make sense to have a `main` entrypoint when you have environment set to `client`.

Reply:
Ahh, _now_ I understood your original comment, you're right, having `main` in a `client` only environment really doesn't make sense.
It's already removed now, so I'll mark this resolved.